constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/MotionLayout.kt

```diff
@@ -654,12 +648,6 @@ fun Transition(@Language("json5") content: String): androidx.constraintlayout.co
     return transition.value
 }
 
-enum class MotionLayoutDebugFlags {
-    NONE,
-    SHOW_ALL,
-    UNKNOWN
-}
```

Comment:
Deprecate instead, to ease transition.

```kotlin
@Deprecated(
   "Use ConstraintSetParser.MotionLayoutDebugFlags instead",
   ReplaceWith("MotionLayoutDebugFlags", "androidx.constraintlayout.core.ConstraintSetParser.MotionLayoutDebugFlags")
)
enum class MotionLayoutDebugFlags {
   ...
}
```

Or if we go the `typealias` route (see next comment)
```kotlin
@Deprecated(
   "Use MotionDebugFlag instead",
   ReplaceWith("MotionDebugFlag", "androidx.constraintlayout.compose.MotionDebugFlag")
)
enum class MotionLayoutDebugFlags
```